src/ucp/api/ucp.h

```diff
@@ -2100,6 +2137,35 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
                                ucp_context_attr_t *attr);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Compare two remote keys according to parameters
+ *
+ * This routine returns equal if all the transports can compare equal and
+ * configurations are equal. At least one compare mode is needed.
```

Comment:
this sentence is not clear

Reply:
removed. proposal was to identify ucp rkeys that would never be subject to compare as equal, so to avoid inserting them into sorted array, in order to avoid pointer-related memory consumption. after discussing with @brminich we now simplify the api making the compare function consistent and we treat all ucp rkey the same.